src/js/store/factory/marks/Scene.ts

```diff
@@ -1,25 +1,43 @@
-'use strict';
-var dl = require('datalib'),
-    Group = require('./Group');
+import {Record, RecordOf} from 'immutable';
+import {BaseMark, Encodable, FromFacet, GroupEncodeEntry, Scope} from 'vega-typings';
+import {Group, LyraGroupMark} from './Group';
 
-/**
- * A scene factory.
- * @returns {Object} Additional default properties for the scene.
- */
-function Scene() {
-  var base = Group();
-  return dl.extend(base, {
-    width: 500,
-    height: 500,
-    padding: 'auto',
-    background: 'white',
-    type: 'scene',
-  });
-}
+const dl = require('datalib');
 
-Scene.getHandleStreams = function() {
-  return {};
-};
+// TODO(jzong): how to keep this in sync with GroupMark in vega-typings?
+export interface LyraSceneMark extends BaseMark, Scope, Encodable<GroupEncodeEntry> {
+  type: 'scene'
+  from?: FromFacet;
```

Comment:
Lyra's `Scene` corresponds to the [top-level of a Vega specification](https://vega.github.io/vega/docs/specification/) which corresponds to the top-level group mark in the resulting scenegraph. As a result, it shouldn't have a `from` property.

Reply:
I'm also confused by the `type: "scene"` but I see it there in the old codebase, so perhaps it means something and some other part of the codebase does something with it? We should mark it as a todo (or perhaps file an issue which will be easier to track) to follow up though.

Reply:
filed issue https://github.com/vega/lyra/issues/434